app/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDao.java

```diff
@@ -89,6 +89,7 @@ public UserConsent getActiveUserConsent(String healthCode, SubpopulationGuid sub
 
         DynamoDBQueryExpression<DynamoUserConsent3> query = new DynamoDBQueryExpression<DynamoUserConsent3>()
             .withScanIndexForward(false)
+            .withLimit(1)
             .withQueryFilterEntry("withdrewOn", new Condition().withComparisonOperator(ComparisonOperator.NULL))
             .withHashKeyValues(hashKey);
```

Comment:
Is it possible for someone to consent, consent again, and then withdraw from the second consent but still have an active first consent? If that's not possible, would it be better to just always get the last consent without the filter, then check if it's withdrawn or not?